lib/fieldFilter.js

```diff
@@ -0,0 +1,40 @@
+'use strict';
+var through = require('through2');
+
+var objectMode = {objectMode: true};
+
+module.exports = function(fields){
+
+  //Chunk is a GeoJSON feature
+  return through(objectMode, function(chunk, enc, cb){
+
+    var payload = {
+      type: "Feature",
+      properties: {},
+      geometry: chunk.geometry
+    }
+
+    var props = chunk.properties;
+
+    for(var i=0; i<fields.length; i++){
+      var field = fields[i];
+      var val;
+
+      if(field.type === 'dynamic'){
+        try{
+          val = new Function('props', field.value)(props); //eslint-disable-line
+        }catch(e){
+          return cb(null);
```

Comment:
Yeah, this is what happens now, part of what has been folded into the large, soon-to-be pull request. The original idea for just returning was that the data count at the end would be sufficient to show that some data was dropped... but pretty much always you'd want to see what that data is, even if it spams your logs